exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,8 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var NumOfTotalPeople = 23;
+
+var percentNum = numberOfStudents*100/NumOfTotalPeople;
```

Comment:
It is good practice to insert a space either side of operators like `*` and `/` - this helps with readability